server/index.js

```diff
@@ -1,16 +1,20 @@
-const express = require("express");
-const morgan = require("morgan");
-const cors = require("cors");
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors');
 const app = express();
+const channel = require('./controller/channel.js');
 // Middleware
 app.use(express.json());
```

Comment:
Can you add a description of what each of the middle does?

Reply:
Yup!